### PR TITLE
Pairplot for arviz

### PR DESCRIPTION
**Pull request by agustinaarroyuelo**
I did the required changes for porting PyMC3's pairplot into Arviz.

### CODE REVIEW

File: arviz/utils/utils.py [hunk 1]
```diff
@@ -182,3 +182,15 @@ def _create_flat_names(varname, shape):
     labels = (np.ravel(xs).tolist() for xs in np.indices(shape))
     labels = (map(str, xs) for xs in labels)
     return ['{}__{}'.format(varname, '_'.join(idxs)) for idxs in zip(*labels)]
+
```
Comment (aloctavodia) [inline arviz/utils/utils.py]: `scale_text` is already used by  `posteriorplot`. Given that you also need it for `pairplot` I suggest to move  `scale_text` inside `pltos/plot_utils.py` and from there import it into `posteriorplot`  and `pairplot`.

File: arviz/plots/pairplot.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
+import warnings
```
Comment (aloctavodia) [inline arviz/plots/pairplot.py]: You are not using this import

**Comment by aloctavodia** [comment]
Thanks @agustinaarroyuelo!!!

**Comment by agustinaarroyuelo** [comment]
You're welcome!

**Pull request merged**